x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -523,6 +523,7 @@ export class EndpointDocGenerator extends BaseDataGenerator {
           entity_id: sessionEntryLeader,
           name: 'fake entry',
           pid: Math.floor(Math.random() * 1000),
+          start: [Date.now().toString()],
```

Comment:
here in below: did you really want `start` to be a localized formatted date? or did you want instead `.toIsoString()`?

Reply:
that is good point Paul. I have updated it.